coverage/artifacts/artsTest/artsTest.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package artsTest stores artifacts functions for tests,
+// used by other packages
+package artsTest
```

Comment:
golang packages _can_ be case sensitive but in practice should not, please don't do this. it will cause surprising issues later. go packages are also directories and many users are on case-insensitive file systems still.

Reply:
see also: https://golang.org/doc/effective_go.html?#package-names

Reply:
removed